src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitedObj = sourceString
+    .split(';')
+    .map((el) => el.trim())
+    .filter((el) => el !== '');
```

Comment:
[CODE KNOWLEDGE] - Don't need to make copy before each operation. Moreover, methods like `filter` or `map` don't mutate original array. There's no need to store the intermediate result in `splitedObj`, you can chain `forEach` directly after `filter`.